packages/trader/src/Modules/Reports/Containers/statement.jsx

```diff
@@ -127,6 +155,7 @@ class Statement extends React.Component {
                     i18n_message={localize(
                         'View all transactions on your account, including trades, deposits, and withdrawals.'
                     )}
+                    optional_component={account_statistics_component}
```

Comment:
I thought we were about to show `account_statistics_component` only for MX clients, but this is shown to all users. Is there a conditional missing?